pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix

```diff
@@ -36,9 +23,24 @@ stdenv.mkDerivation rec {
     gst_all_1.gst-plugins-base
     gst_all_1.gst-plugins-good
     gtkmm3
-    libappindicator-gtk3
+    libayatana-appindicator
+    libcanberra
+    libdatrie
+    libdeflate
+    libepoxy
+    libpsl
+    libselinux
+    libsepol
+    libsysprof-capture
+    libthai
+    libxkbcommon
     pcre
+    pcre2
```

Comment:
pcre and pcre2?

Reply:
Because when excluded the package build complains about:
```
Package 'libpcre', required by 'libselinux', not found
```

Reply:
And when pcre2 is excluded it complains about:
```
Package 'libpcre2-8', required by 'glib-2.0', not found
```